Automattic/sensei
# Add No Progress Reminder emails

Resolves part of #6624

## Proposed Changes

* Add patterns and seeder data for No Progress reminders.

## Testing Instructions

1. Enable `email_customization` feature flag.
2. Recreate Emails in `Sensei LMS -> Tools`.
3. Check that No Progress Reminder emails appear in the list: `Sensei LMS -> Settings -> Emails`
4. Check Sensei Pro for the [counterpart PR](https://github.com/Automattic/sensei-pro/pull/2198)

## Screenshots
<img width="1678" alt="CleanShot 2023-03-17 at 05 25 37@2x" src="https://user-images.githubusercontent.com/329356/225766837-5ef4f09b-93cd-462d-ad0c-611d10cb88e7.png">


## Pre-Merge Checklist
